source/css/sections/_contents.scss

```diff
@@ -111,4 +111,8 @@
       flex-basis: 17.5%; // Fit into fifths, but account for padding
     }
   }
+
+  img { 
+    max-width: 100%;
```

Comment:
@kylorhall this was the addition that fixed the not wrapping images Angela was talking about [here](https://sharesight.slack.com/archives/C01EKA6R04T/p1651212064103089). I have a sneaky suspicion that some of the blog styles that were removed were causing this to be applied to all images but I couldn't get middleman working locally (or not even sure that is possible?) so couldn't confirm exactly.

Reply:
also removed this - not needed after readding some of the global styles that were removed